hack/install-deps.sh

```diff
@@ -52,6 +52,8 @@ CNI_CONFIG_DIR=${DESTDIR}/etc/cni/net.d
 RUNC_PKG=github.com/opencontainers/runc
 CNI_PKG=github.com/containernetworking/plugins
 CONTAINERD_PKG=github.com/containerd/containerd
+CRICTL_PKG=github.com/kubernetes-incubator/cri-tools
+CRICTL_VERSION=b3c7579714aedaf9185624ab9ca92faefc73b56b
```

Comment:
Use `hack/versions`, so that we don't need to update it everywhere.

You may want to change the version name to `CRITOOLS_VERSION`, so that it includes both critest and crictl.

Reply:
sure makes sense.